x-pack/plugins/features/server/oss_features.test.ts

```diff
@@ -5,6 +5,8 @@
  */
 
 import { buildOSSFeatures } from './oss_features';
+import { featurePrivilegeIterator } from '../../security/server/authorization';
```

Comment:
🙈 import from non-dependant plugin + import from nested path (why doesn't eslint complains about that last point btw?). Maybe `featurePrivilegeIterator` belongs more in the `features` plugin?

Reply:
Yeah.... we have a discussion around this here: https://github.com/elastic/kibana/pull/57507#discussion_r384227385.

I'm torn on what to do. The current situation is not at all ideal..We only ever expect `featurePrivilegeIterator` to be consumed in two scenarios:
1) internal to the security plugin, as it has knowledge of how the privileges are merged together.
2) external to the security plugin as part of another plugin's test suite only. We wanted to provide a mechanism for plugin authors to write tests against their feature registration, to ensure their privileges are created in the way they expect.

I'm not opposed to moving this to the features plugin, but I don't love that we'd be moving security-specific logic out of security and into features. What do you think?

Reply:
If the only usages outside of the security plugin are meant to be for testing purposes, maybe exposing it from a `security/server/test_utils` or something would be more explicit? 

Note that if we do have conventions about mocks and `plugin_name/[server|public]/mocks`, do we not have the equivalent for exposing test utilities to other plugins atm.

This can be done at a later time though, not a blocker here.

ping @restrry @joshdover

Reply:
not Ideal, but we can expose test utils from `mocks`